tests/qos/test_buffer.py

```diff
@@ -933,10 +933,17 @@ def port_to_test(request, duthost):
             "LAG member port {} can not be used for dynamic buffer test".format(PORT_TO_TEST))
         PORT_TO_TEST = None
     if not PORT_TO_TEST:
-        PORT_TO_TEST = list(testPort)[0]
-    lanes = duthost.shell(
-        'redis-cli -n 4 hget "PORT|{}" lanes'.format(PORT_TO_TEST))['stdout']
-    NUMBER_OF_LANES = len(lanes.split(','))
+        # The NVIDIA SPC1 platform requires a 4 lanes port for testing to avoid exceeding the maximum available headroom
```

Comment:
is this limitation is only for SN2700 or might be relevant for other platforms as well?

Reply:
> is this limitation is only for SN2700 or might be relevant for other platforms as well?

Confirmed with Stephen that this limitation is only for SN2700 because the maximum headroom values ​​are different for different platforms. The maximum for SN2700 is much smaller so it needs special handling